.env

```diff
@@ -0,0 +1,8 @@
+#PORT=8000
```

Comment:
Can we get rid of this `.env` file and just provide an example dockerfile with values that can be edited?

Reply:
.env file is optional, but good practice - users should edit that instead of the Compose file directly